tests/e2e-openshift/otel-multitenancy-smoketest/02-generate-traces.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1
```

Comment:
For this step, I would set a high nunmber for the duration and just assert the next step (verify the traces were received)

Reply:
Do you suggest removing `02-assert.yaml`? I think it's good to know if the traces got generated successfully, and only after that verify that the traces are there. Afaics the Job has retry built in (via the Kubernetes Job object), but in the normal case there shouldn't be any retry, if we wait until the previous job (generate traces) was successful.